src/diffusers/schedulers/scheduling_deis_multistep.py

```diff
@@ -449,7 +486,24 @@ def _convert_to_exponential(self, in_sigmas: torch.Tensor, num_inference_steps:
     def _convert_to_beta(
         self, in_sigmas: torch.Tensor, num_inference_steps: int, alpha: float = 0.6, beta: float = 0.6
     ) -> torch.Tensor:
-        """From "Beta Sampling is All You Need" [arXiv:2407.12173] (Lee et. al, 2024)"""
+        """
+        Construct a beta noise schedule as proposed in ["Beta Sampling is All You
+        Need"](https://huggingface.co/papers/2407.12173) (Lee et al., 2024).
```

Comment:
Make this change to the other `_convert_to_beta` docstrings as well :)

```suggestion
        Construct a beta noise schedule as proposed in [Beta Sampling is All You
        Need](https://huggingface.co/papers/2407.12173).
```